data/participants.json

```diff
@@ -11,9 +11,9 @@
             "referral" : ""
         },
 	 {
-            "name"     : "Archies11",
-			"imageurl" : "assets/img/participants/archisha.jpg",
-			"about"    : "Curious mind keen to learn",
+            "name"     : "agentprotn",
```

Comment:
Another such offence may lead to your disqualification.

@pradeepgangwar @Anupam-dagar

Reply:
@zerefwayne ,i got my mistake